File: frontend/src/DTO/response/CompletedExerciseResponse.ts
import type { ExerciseEnum } from '@/enums/ExerciseEnum.ts'
import type { DifficultyEnum } from '@/enums/DifficultyEnum.ts'

export type CompletedExerciseResponse = {
  id: number
  exercise: ExerciseEnum,
  earnedPoints: number,
  difficulty: DifficultyEnum,
  completedAt: string
}

File: backend/src/main/java/vr/balance/app/DTO/response/CompletedExerciseResponse.java
package vr.balance.app.DTO.response;

import lombok.Data;
import vr.balance.app.enums.DifficultyEnum;
import vr.balance.app.enums.ExerciseEnum;

import java.time.Instant;

@Data
public class CompletedExerciseResponse {
    ExerciseEnum exercise;
    int earnedPoints;
    Instant completedAt;
    DifficultyEnum difficulty;
}
